File: src/scala/org/codehaus/testdox/intellij/actions/ActivateTestDoxPanelAction.scala
package org.codehaus.testdox.intellij.actions

import com.intellij.openapi.actionSystem.AnActionEvent

import org.codehaus.testdox.intellij.IconHelper
import org.codehaus.testdox.intellij.IconHelper._

class ActivateTestDoxPanelAction extends BaseAction("Toggle TestDox", "Shows/hides the TestDox panel for the current class", getIcon(IconHelper.TESTDOX_ICON)) {

  def actionPerformed(event: AnActionEvent) {
    actionEvents.getTestDoxController(event).toggleToolWindow()
  }
}

File: src/java/org/codehaus/testdox/intellij/actions/ActivateTestDoxPanelAction.java
package org.codehaus.testdox.intellij.actions;

import com.intellij.openapi.actionSystem.AnActionEvent;

import org.codehaus.testdox.intellij.IconHelper;

public class ActivateTestDoxPanelAction extends BaseAction {

    public ActivateTestDoxPanelAction() {
        super("Toggle TestDox", "Shows/hides the TestDox panel for the current class", IconHelper.getIcon(IconHelper.TESTDOX_ICON));
    }

    public void actionPerformed(AnActionEvent event) {
        actionEvents.getTestDoxController(event).toggleToolWindow();
    }
}
